pupil_src/shared_modules/surface_tracker/surface_tracker_offline.py

```diff
@@ -337,6 +337,21 @@ def _start_gaze_buffer_filler(self, all_gaze_events, all_world_timestamps, secti
             mp_context,
         )
 
+    def _start_fixation_buffer_filler(
```

Comment:
I removed that because I could not find any usage of this function. In case this function is needed, fixations_on_surf_buffer_filler needs to be defined somewhere outside of this function or the if condition raises an attribute error.